manifests/master.machineconfigpool.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     "operator.machineconfiguration.openshift.io/required-for-upgrade": ""
     "machineconfiguration.openshift.io/mco-built-in": ""
+    pools.operator.machineconfiguration.openshift.io/master: ""
```

Comment:
for consistency, and because I don't trust yaml, should we put the key in quotes like the other labels?